source/standards/source-code.html.md.erb

```diff
@@ -15,7 +15,7 @@ At GDS, we follow the principles set out in the Service Manual for managing the
 
 Wherever possible, we [make our source code open and reusable][]. This means other government departments and people in outside organisations can benefit from our work. We also maintain several open source projects developed for use on GOV.UK and with other work we do, such as the [Puppet module for go-audit][] or the [GDS Operations open source web site][].
 
-It’s not always appropriate to open code. There are situations [when you should keep some data and code closed][], for example:
+It’s not always appropriate to open code. There are sometimes grounds for [keeping some data and code closed][], for example:
```

Comment:
This link is missing it's URL?

I think you need to update the `when you should keep some data and code closed` bit further down.